lib/src/main/java/io/ably/lib/realtime/ChannelProperties.java

```diff
@@ -0,0 +1,15 @@
+package io.ably.lib.realtime;
```

Comment:
I think I would have put this into `types`, in case in the future we use it also for Rest channels